pkg/domain/infra/abi/system.go

```diff
@@ -33,42 +30,6 @@ func (ic *ContainerEngine) Info(ctx context.Context) (*define.Info, error) {
 	return ic.Libpod.Info()
 }
 
-func (ic *ContainerEngine) RestService(_ context.Context, opts entities.ServiceOptions) error {
-	var (
-		listener *net.Listener
-		err      error
-	)
-
-	if opts.URI != "" {
-		fields := strings.Split(opts.URI, ":")
-		if len(fields) == 1 {
-			return errors.Errorf("%s is an invalid socket destination", opts.URI)
-		}
-		address := strings.Join(fields[1:], ":")
-		l, err := net.Listen(fields[0], address)
-		if err != nil {
-			return errors.Wrapf(err, "unable to create socket %s", opts.URI)
-		}
-		listener = &l
-	}
-
-	server, err := api.NewServerWithSettings(ic.Libpod, opts.Timeout, listener)
-	if err != nil {
-		return err
-	}
-	defer func() {
-		if err := server.Shutdown(); err != nil {
-			logrus.Warnf("Error when stopping API service: %s", err)
-		}
-	}()
-
-	err = server.Serve()
-	if listener != nil {
-		_ = (*listener).Close()
-	}
-	return err
-}
-
 func (ic *ContainerEngine) VarlinkService(_ context.Context, opts entities.ServiceOptions) error {
 	var varlinkInterfaces = []*iopodman.VarlinkInterface{
 		iopodmanAPI.New(opts.Command, ic.Libpod),
```

Comment:
```suggestion
func (ic *ContainerEngine) VarlinkService(_ context.Context, opts entities.ServiceOptions) error {
	var varlinkInterfaces = []*iopodman.VarlinkInterface{
		iopodmanAPI.New(opts.Command, ic.Libpod),
```
Should this move into cmd/podman as well?

Reply:
Yes, this would be another nice candidate. For now, this PR focuses on `podman rmi`.